src/main/webapp/renderers/RendererInterface.js

```diff
@@ -0,0 +1,9 @@
+export default class RendererInterface {
+    constructor(parentElement) {
+        this.parentElement = parentElement;
+    }
+
+    render() {
+        
```

Comment:
Is something going to go in here? If not do {} rather than the newline

Reply:
I want this to be like a java interface (to let people know about this signature of the render classes), but it ended up being an abstract class. Is this the right way to go? Can you suggest a better way to do this?